src/ripple/app/ledger/LedgerTiming.h

```diff
@@ -130,6 +130,14 @@ auto constexpr AV_STUCK_CONSENSUS_PCT = 95;
 
 auto constexpr AV_CT_CONSENSUS_PCT = 75;
 
+// The minimum amount of time to consider the previous round
+// to have taken. This ensures that there is an opportunity
+// for a round at each avalanche threshold even if the
+// previous consensus was very fast. This should be at least
+// twice the interval between proposals (0.7) divided by
+// the interval between mid and late consensus ([85-50]/100).
```

Comment:
I really don't understand how to turn `([85-50]]/100)` into an interval.  That's cool, if the comment is primarily for yourself.  But as it stands the comment doesn't say quite enough to be useful for me.